pkg/trait/jvm.go

```diff
@@ -205,8 +205,11 @@ func (t *jvmTrait) enableDebug(e *Environment) string {
 
 func (t *jvmTrait) prepareClasspathItems(container *corev1.Container) []string {
 	classpath := sets.NewSet()
+	// Depreacted
```

Comment:
typo: Deprecated. Also, it would be good to provide an alternative or what we'll expect to do in the future (ie, won't be supported any longer).